jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -305,7 +327,8 @@ boolean isRPCServer() {
               startTimeMicroseconds,
               startTimeNanoTicks,
               computeDurationViaNanoTicks,
-              tags);
+              tags,
+              references);
```

Comment:
If `parent_id` only used to maintain compatibility with Zipkin's model (as mentioned by @yurishkuro) - then the other alternative is Zipkin's parent id is simply converted to a `ChildOf` reference - then the new model would be "pure" :)